common/src/main/cpp/extensions/openxr_fb_scene_extension_wrapper.cpp

```diff
@@ -34,10 +34,6 @@
 #include <godot_cpp/templates/local_vector.hpp>
 #include <godot_cpp/variant/utility_functions.hpp>
 
-#ifdef META_VENDOR_ENABLED
-#include <openxr/fb_scene.h>
-#endif
```

Comment:
You can also remove the `#ifdef META_VENDOR_ENABLED` guard on line 110.

Reply:
Done!